packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -254,20 +217,21 @@ function HeaderBar(props) {
 		Intercom,
 	});
 
+	if (props.brand && props.products) {
+		console.warn('Deprecated: use @talend/ui-ee/AppSwitcher');
+	}
+
+	const AppSwitcherComponent =
+		props.AppSwitcher || Inject.get(props.getComponent, 'AppSwitcher', AppSwitcher) || null;
```

Comment:
```suggestion
	const AppSwitcherComponent =
		props.AppSwitcher || Inject.get(props.getComponent, 'AppSwitcher', AppSwitcher);
```

The Inject will always give you at least the default AppSwitcher component